src/common/components/repositories-list/index.js

```diff
@@ -119,14 +129,25 @@ class RepositoriesList extends Component {
   }
 }
 
-RepositoriesList.propTypes = {
+RepositoriesListView.defaultProps = {
```

Comment:
Decouples the component a little more from store, and makes for fewer fixtures in tests